test/ToricVarieties/line_bundles.jl

```diff
@@ -1,36 +1,46 @@
-@testset "Toric line bundles" begin
-    dP3 = NormalToricVariety([[1, 0], [1, 1], [0, 1], [-1, 0], [-1, -1], [0, -1]], [[1, 2], [2, 3], [3, 4], [4, 5], [5, 6], [6, 1]])
-    F5 = NormalToricVariety([[1, 0], [0, 1], [-1, 5], [0, -1]], [[1, 2], [2, 3], [3, 4], [4, 1]])
-    D2 = DivisorOfCharacter(F5, [1, 2])
-    dP1 = NormalToricVariety([[1, 0], [0, 1], [-1, 0], [-1, -1]], [[1, 2], [2, 3], [3, 4], [4, 1]])
+using Oscar
+using Test
 
+@testset "Line bundles (set_attributes = $set_attributes)" for set_attributes in [true, false]
+    
+    dP1 = del_pezzo_surface(1; set_attributes)
+    dP3 = del_pezzo_surface(3; set_attributes)
+    
     l = ToricLineBundle(dP3, [1, 2, 3, 4])
-    l2 = ToricLineBundle(D2)
-    l3 = ToricLineBundle(dP1, [fmpz(1), fmpz(2)])
-    l4 = canonical_bundle(dP3)
-    l5 = anticanonical_bundle(dP3)
-    l6 = ToricLineBundle(dP3, trivial_divisor(dP3))
-    l7 = structure_sheaf(dP3)
-
+    l2 = canonical_bundle(dP3)
+    l3 = anticanonical_bundle(dP3)
+    l4 = ToricLineBundle(dP3, trivial_divisor(dP3))
+    l5 = ToricLineBundle(dP1, [fmpz(1), fmpz(2)])
+    
     @testset "Should fail" begin
-        @test_throws ArgumentError l * l3
+        @test_throws ArgumentError l * l5
     end
-
+    
     @testset "Basic properties" begin
         @test is_trivial(l) == false
         @test is_basepoint_free(l) == false
         @test is_ample(l) == false
         @test is_very_ample(l) == false
-        @test degree(l) == 10
+    end
+    
+    @testset "Basic attributes" begin
+        if set_attributes
+            @test degree(l) == -6
+            @test degree(l^(-1)) == 6
+            @test degree(l*l) == -12
+        else
+            @test degree(l) == 10
+            @test degree(l^(-1)) == -10
+            @test degree(l*l) == 20
+        end
```

Comment:
We should avoid having too many of these blocks. Could you maybe add a short description as a comment above that the rays have different order and hence the divisors are different?

Reply:
I send you more details in slack. Maybe best to discuss there.

Long story short: You are right, I need to explain this in more detail. I shall do so in a separate PR towards the end of the upcoming week, i.e. once the big renaming is complete.

Reply:
With that said, I will merge this PR now.